src/dfmt/ast_info.d

```diff
@@ -85,6 +103,8 @@ struct ASTInformation
 
     /// Locations of constructor/destructor "this" tokens ?
     size_t[] constructorDestructorLocations;
+
+    LocationPair[] sortedByEndLocation;
```

Comment:
This should probably be called something like `braceIndentInfo` because it contains information about the indentation of open/close brace pairs instead of information about sorting by end locations.